src/torrent.cpp

```diff
@@ -102,6 +102,7 @@ POSSIBILITY OF SUCH DAMAGE.
 #endif
 
 using namespace std::placeholders;
+using namespace std::chrono;
```

Comment:
generally it's better to just pull in the names you need, like this:
using std::chrono::duration_cast;
using std::chrono::seconds;